packages/core/src/components/legend-item/legend-item.scss

```diff
@@ -0,0 +1,60 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/typography";
+@use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/components/legend-item/legend-item.mixins" as legend-item-mixins;
+@use "~dso-toolkit/src/components/legend-item/legend-item.variables" as legend-item-variables;
+
+:host {
+  display: block;
+  padding: 5px;
+}
+
+@include utilities.box-sizing();
+
+.sr-only {
+  @include utilities.sr-only();
+}
+
+.dso-legend-item {
+  display: flex;
+  position: relative;
+
+  &.dso-legend-item-symbol {
+    + .dso-body {
+      margin-inline-start: legend-item-variables.$symbol-width + units.$u1;
+    }
+  }
+
+  &.dso-legend-item-selectable {
+    + .dso-body {
+      margin-inline-start: legend-item-variables.$selectable-width;
+    }
+
+    &.dso-legend-item-symbol {
+      + .dso-body {
+        margin-inline-start: legend-item-variables.$symbol-width + units.$u1 + legend-item-variables.$selectable-width;
+      }
+    }
+  }
+
+  button {
+    background: none;
+    border: 0;
+    border-radius: 0 #{units.$u1 * 0.5} #{units.$u1 * 0.5} 0;
+    color: inherit;
+    float: right;
```

Comment:
Is `float` niet een beetje uit de tijd (en onhandig ivm de noodzaak om te clearfixen)? volgens mij is het handiger om hier flexbox te gebruiken.

Reply:
Kijk er samen even naar, volgens mij kan het inderdaad simpeler :+1:

Reply:
Ik hanteer dezelfde styling als voor het label component. Als we vinden dat float niet meer kan, wil ik voorstellen om dat buiten deze PR te houden en daar een separaat issue voor aan te maken. Of denken jullie @tfrijsewijk en @hansgrimm daar faliekant anders over.

Reply:
Op zich eens, we hebben zelfs al een issue die ook dit onderwerp (deels) raakt: #2260